docs/guides/add-exposure-tests.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: "Add exposure validation tests"
+---
+
+After you [install the dbt package](/quickstart#install-the-dbt-package), you can add Elementary exposure validation tests.
+
+## Exposure validation dbt tests
+
+Elementary dbt package includes **exposure validation tests, implemented as [dbt tests](https://docs.getdbt.com/docs/building-a-dbt-project/tests)**.
+These tests can test that there are column changes that break downstream exposures, such as BI dashboards.
+
+## Configure your elementary exposure validation tests
+
+Within your `models` directory, add a file called `exposures.yml`
+
+```yml
+version: 2
+exposures:
+  - name: customers
+    label: Customer Dashboard
+    type: dashboard
+    maturity: high
+    url: https://your.bi.tool/dashboards/1
+    description: >
+      Shows customer growth
+
+    depends_on:
+      - ref('customers')
+
+    owner:
+      name: Callum McData
+      email: data@jaffleshop.com
+```
+
+<Tip>
+  You can read up about exposures and how to add them at [the dbt docs
+  site](https://docs.getdbt.com/docs/build/exposures)
+</Tip>
+
+### Adding the Elementary exposure information
+
+For each exposure wish to be verified, you add a new property called `meta`:
+
+```yml
+version: 2
+exposures:
+  - name: customers
+    label: Customer Dashboard
+    type: dashboard
+    maturity: high
+    url: https://your.bi.tool/dashboards/1
+    description: >
+      Shows customer growth
+
+    depends_on:
+      - ref('customers')
+
+    owner:
+      name: Callum McData
+      email: data@jaffleshop.com
+
+    meta:
+      referenced_columns:
+        - name: "customer_id"
+```
+
+<Tip>
+  In the Elementary Cloud, the exposures and exposure validation tests are added
+  automatically, according to the actual dependency if your BI tool.
+</Tip>
+
+You can optionally also specify the column data type (`data_type`)
+
+```yml
+version: 2
+- name: returned_orders
+  label: Returned Orders
+  type: dashboard
+  maturity: high
+  url: https://your.bi.tool/dashboards/2
+  description: >
+    Returned orders over time
+
+  depends_on:
+    - ref('returned_orders')
+
+  owner:
+    name: Callum McData
+    email: data@jaffleshop.com
+  meta:
+    referenced_columns:
+      - name: "order_id"
+        data_type: "numeric"
+```
+
+In addition, if your exposure depends on several nodes, you can depend explicitly per column which source should be tested:
+
+```yml
+version: 2
+- name: returned_orders
+  label: Returned Orders
+  type: dashboard
+  maturity: high
+  url: https://your.bi.tool/dashboards/2
+  description: >
+    Returned orders over time
+
+  depends_on:
+    - ref('returned_orders')
+    - ref('orders')
+
+  owner:
+    name: Callum McData
+    email: data@jaffleshop.com
+  meta:
+    referenced_columns:
+      - name: "order_id"
+        data_type: "numeric"
+        source: ref('returned_orders')
+      - name: "customer_id"
+        data_type: "numeric"
+        source: ref('orders')
```

Comment:
Not sure it's possible, but it will be nice if we could do something like:
referenced_columns:
  ref('orders'):
    - name: "order_id"
      data_type: "numeric"
    - name: "customer_id"
      data_type: "numeric"
  ref('customers'):
    - ...

Reply:
not sure, and it'll make the "simple" workflow (AKA no need to set the source property since there's only one exposure dependency) obsolute, which IMHO is a shame, it's prolly our 80 vs 20 scenario

Reply:
Pushed, only difference is `node` instead of `model_name` as per our conversation and @haritamar 's comment on the main PR